src/main/docker/Dockerfile-centos.aarch64

```diff
@@ -0,0 +1,36 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# ActiveMQ Artemis
+
+FROM arm64v8/centos:8 as builder-env
+
+LABEL maintainer="Apache ActiveMQ Team"
+
+RUN yum groupinstall -y "Development Tools" && \
+	yum install -y vim cmake libaio libaio-devel java-1.8.0-openjdk-devel && yum clean all
+
+FROM builder-env as builder
+
+ADD . /work
+
+VOLUME ["/work/target/lib"]
+VOLUME ["/root/.m2/repository"]
+
+WORKDIR /work
+
+CMD [ "sh", "-c", "/work/scripts/compile-native.sh"]
```

Comment:
At the moment `compile-native.sh` looks like:

```bash
# This will compile both 32 and 64 bits version

cmake -DCMAKE_VERBOSE_MAKEFILE=On -DCMAKE_USER_C_FLAGS="-m32" -DARTEMIS_CROSS_COMPILE=On -DARTEMIS_CROSS_COMPILE_ROOT_PATH=/usr/lib .
make
rm -rf CMakeCache.txt cmake_install.cmake
cmake -DCMAKE_VERBOSE_MAKEFILE=On -DCMAKE_USER_C_FLAGS="" -DARTEMIS_CROSS_COMPILE=Off -DARTEMIS_CROSS_COMPILE_ROOT_PATH=/usr/lib .
make
```

and it seems to work just fine.
If needed we can pass argument(s) whether to build i686, x64 or aarch64.